scripts/tests/chiptest/test_definition.py

```diff
@@ -142,30 +145,42 @@ def __startServer(self):
                     self.kvsPathSet.add(value)
         return self.runner.RunSubprocess(app_cmd, name='APP ', wait=False)
 
-    def __waitFor(self, waitForString, server_process, outpipe, timeoutInSeconds=10):
-        logging.debug('Waiting for %s' % waitForString)
+    def __waitFor(self, *symptoms, **kwargs):
+        """
+        Wait for all provided symptom strings to appear in the capture log.
+        """
+        server_process = kwargs.get('server_process', self.process)
+        outpipe = kwargs.get('outpipe', self.outpipe)
+        timeoutInSeconds = kwargs.get('timeoutInSeconds', 10)
```

Comment:
Should we maybe kave kwargs be something like `server_process=None, outpipe=None` and such? 

If we use full **kwargs, this is really hard to self-document.
Also can we have symptoms be `list[str]` and pass it as such? That way looking at the signature is self-describing and static linters will check types.